pkg/client/manifest_client.go

```diff
@@ -22,11 +22,16 @@ type CreateExternalSecretOpts struct {
 // ManifestService implements the business logic
 type ManifestService interface {
 	CreateExternalSecret(ctx context.Context, opts CreateExternalSecretOpts) (*ExternalSecret, error)
+	// I don't like this, but I think ManifestService is an equally big problem
```

Comment:
Can you specify what "this" means, so the next developer has a better idea what the problem is?

Reply:
Good point, done!